pages/learn/json-schema-examples.md

```diff
@@ -92,129 +177,403 @@ title: JSON Schema examples
 }
 ```
 
-## Card
+**Data**
+
 ```json
 {
-  "$id": "https://example.com/card.schema.json",
+  "startDate": "2023-08-25T10:00:00Z",
+  "endDate": "2023-08-25T12:00:00Z",
+  "summary": "Conference Presentation",
+  "location": "Conference Center",
+  "recurrenceDule": "FREQ=DAILY;COUNT=5"
+}
+```
+
+
+## Device type
```

Comment:
I'm not sure what you're trying to do with this example.

The use of `$dynamicRef` is wrong,

- In 2020-12, `$dynamicRef` needs a `$dynamicAnchor` defined in the same schema resource, otherwise it works like an ordinary `$ref`.
- The value of `$dynamicRef` should be an anchor fragment (e.g. `#foo`), hence needing the anchor defined.
- The location `#/$defs/DeviceTypes` doesn't exist.

And I'm not sure a dynamic reference is the best solution for what the description says.  I'd use

- `oneOf` with `deviceType` defined using a `const` with a different value in each of the subschemas
- `if`/`then`/`else`
- follow [this example](https://json-schema.org/blog/posts/modelling-inheritance) to model inheritance (complex/advanced)